static_inventories/betabarrel_cluster.yml

```diff
@@ -66,7 +69,6 @@ cluster:
 betabarrel_cluster:
   children:
     openstack_api:
-    jumphost:
```

Comment:
This will not go. If we remove jumphost group from static inventory, then playbook will fail.

Reply:
Is there a special reason why we would need to remove it?

Reply:
Hmmm, Ok that sucks, but I guess it is reasoble to expect every stack to have a jumphost. The problem was in the lor-config function, that fgreps the value for jumphost from the static inventory and picks up the wrong machine name if jumphost is mentioned here.

Reply:
Well, I just learned to manually specify a specific jumphost for this two machines and that's all.
`export AI_PROXY=porch`
It's not a lot. And it is a nice way to practice this flexibility of selecting custom jumphost. As long as the jumphost has actual access to the machine.

Reply:
You will still need to do that manually, but instead of configuring a wrong value the ```lor-config``` command will now:
```
INFO: Using AI_PROXY:     None: no jumphost specified in ANSIBLE_INVENTORY
```

Reply:
I tested it, and it works :+1: